assets/sass/modules/_consent.scss

```diff
@@ -109,16 +43,20 @@
     }
   }
 
+  .scope-list {
+    padding-bottom: $space-between-container;
+  }
+
   .consent-description {
-    padding-top: 15px;
-    padding-bottom: 25px;
-    font-size: $body-font-size;
+    padding-bottom: $space-of-last-container;
```

Comment:
Just curious, how can we tell that this element with class of `.consent-description` should get padding-bottom of $space-of-last-container where as `.scope-list` gets a padding-bottom of $space-between-container. Would it make sense to add another class to `consent-description` something like `last-container`?

Reply:
we could go that route then we could 
- add generic class like `consent-row` to all those sections
- add css like 
    - `consent-row (not last) {padding: 25}`
    - `consent-row (last) {padding: 14}`

Reply:
That seems ideal. Is that a small change to do?

Reply:
Addressed.